### PR TITLE
Index page statistics

### LINKED ISSUE #63: User Story 15, Items Index Page Statistics
```
As any kind of user on the system
When I visit the items index page ("/items")
I see an area with statistics:
- the top 5 most popular items by quantity purchased, plus the quantity bought
- the bottom 5 least popular items, plus the quantity bought

"Popularity" is determined by total quantity of that item fulfilled
```

DO THESE STORIES FIRST:
- User Story 14, Items Index Page

### Checklist:

- [ ] base tests are written
- [ ] base code is written to pass tests
- [ ] edge case ("sad path") tests written (if required)
- [ ] edge case code written (if required)
- [ ] code is reviewed
- [ ] branch is merged
- [ ] all tests still pass

child of #65

child of #82

depends on #64

### PR DESCRIPTION
**Pull request by renecasco**
- [] Check this if the PR has been approved by the team to be a quick patch and the rest of this form will not be filled out

# Test Coverage Percentage (Model/Feature):
- rspec - 100%
- rspec spec/models - 100%

# What functionality does this accomplish?
5 most popular and 5 least popular items stats

# What did you struggle on to complete?
test setup data

# Implements/Fixes:
* description
closes #63 

# Testing Changes
- [x] No Tests have been changed
- [] Some Tests have been changed
- [] All of the Tests have been changed(Please describe what in the world happened)

# Checklist:

- [x] My code has no unused/commented out code
- [x] I have reviewed my code
- [] I have commented my code, particularly in hard-to-understand areas
- [x] I have fully tested my code

# Please include an emoji of how you feel about this branch:
💯

### CODE REVIEW

File: spec/models/item_spec.rb [hunk 1]
```diff
@@ -42,7 +41,7 @@
```
Comment (blake-enyart) [inline spec/models/item_spec.rb]: Could we get some comments in this section of code for future reference?
